testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorTest.java

```diff
@@ -2157,6 +2157,78 @@ public void testDropColumn()
         assertFalse(getQueryRunner().tableExists(getSession(), tableName));
     }
 
+    @Test
+    public void testDropFieldFromRowType()
+    {
+        skipTestUnless(hasBehavior(SUPPORTS_DROP_COLUMN) && hasBehavior(SUPPORTS_ROW_TYPE));
```

Comment:
Let's define 
```
SUPPORTS_DROP_FIELD(fallback -> fallback.test(SUPPORTS_DROP_COLUMN) && fallback.test(SUPPORTS_ROW_TYPE)),
```

(this requires moving SUPPORTS_ARRAY, SUPPORTS_ROW_TYPE up; the types could go at the top of the enum)